src/configuration/advanced/system.md

```diff
@@ -214,5 +214,16 @@ _Enhanced Media Gallery_
 |--- |--- |--- |
 |Enabled|Global|Enables or disables the new Media Gallery.|
 
+## Media Gallery Image Optimization
+
+![Advanced configuration - Media Gallery Image Optimization]({% link images/images/enhanced-media-gallery/system-image-optimization.png %}){: .zoom}
+_Media Gallery Image Optimization_
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enable Image Optimization|Global|Resize images to decrease the file size of images inserted to the content. Original images are preserved in Media Gallery.|
+|Maximum Width|Global|The size of image inserted from Media Gallery to the content will be reduced to match this value.|
+|Maximum Height|Global|The size of image inserted from Media Gallery to the content will be reduced to match this value.|
+
```

Comment:
I am assuming that aspect ratio is maintained, so that the image is reduced as much as needed to fall at or below these max values.